doc/scenarioParametersConfiguration.md

```diff
@@ -267,11 +267,72 @@ parameters:
           acceptsEmptyFields: true
 ```
 
-#### Notes
+Notes:
 
 - defining several times the same field will result in undefined behavior
 - setting different value types in the `type` attribute will result in undefined behavior
 
+#### Columns grouping definition
+
+The description of the table groups consists of putting columns in the same group. Each group needs to have this following fields:
```

Comment:
Typo: "this  following fields" -> "the following fields"

Reply:
DONE